frontend/src/components/BestSellersSection.tsx

```
@@ -1,23 +1,68 @@
+        <div className="flex gap-2">
+          {Array.from({ length: Math.ceil(products.length / 3) }).map((_, index) => (
+            <button
+              key={index}
+              onClick={() => setCurrentIndex(index * 3)}
+              className={`w-2 h-2 rounded-full transition-all ${
+                currentIndex === index * 3
+                  ? 'bg-black w-8'
+                  : 'bg-gray-300 hover:bg-gray-400'
+              }`}
+              aria-label={`Ver grupo ${index + 1}`}
+            />
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)